docs-src/index.md

```diff
@@ -1,36 +1,52 @@
 # Introduction
 
-Service APIs is the evolution of Kubernetes APIs that relate to `Services`,
-e.g. `Ingress`. The project is part of Kubernetes, working under
-[SIG-NETWORK][sig-network]. Use the following resources to learn more about
-Service APIs.
+Service APIs is an open source project, working under
+[SIG-NETWORK][sig-network], that is evolving service networking APIs within the Kubernetes ecosystem. Service APIs are the APIs you use to expose your Kubernetes applications - Services, Ingress, and more. The Gateway API is the primary API being developed to unify service networking under a core set of networking APIs.
+
+### What is the Gateway API?
+
+The Gateway API is an effort by a consortioum of companies and users to evolve the current Kubernetes Ingress API. Gateway aims to improve upon Ingress by becoming a more expressive, extensible, role-oriented API while remaining a generic routing API that has many implementations and broad industry support. 
+
+Gateway is really a collection of API resources - `GatewayClass`, `Gateway`, `HTTPRoute`, `TCPRoute`, and so on. These resources are used together to model a wide variety of different networking use-cases. See [API Concepts](concepts.md) to learn more.
+
+![Gateway API Model](./images/api-model.png)
```

Comment:
Similar comments around naming here, I think we should be using "Service APIs" wherever possible for consistency.

Reply:
I changed it to using "Service APIs" in a more generic way. It does feel strange to not have a more concrete name for the project though because "Service APIs" feels like such an amorphous reference for what we're talking about, but that's a discussion for another day :)